sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/AzureMonitorTransmitter.cs

```diff
@@ -125,14 +120,19 @@ public async ValueTask TransmitFromStorage(long maxFilesToTransmit, bool async,
 
                         if (result == ExportResult.Success)
                         {
-                            blob.Delete();
+                            blob.TryDelete();
```

Comment:
Only if this returned `true` should the next log a success?

Reply:
No - success should be logged based on result.
If the delete fails for some reason, the same record could be transmitted twice.

Reply:
Swapping the lines, should help read the code easily.